spec/integrations/discord/interactions/resolvers/message_spec.rb

```diff
@@ -72,35 +69,44 @@
       it 'sets the correct expiration type on the message' do
         expect(Message.last.expiration_type).to eq(context.option_value('expiration_type'))
       end
-    end
-
-    context 'when message creation fails' do
-      let(:messages_creator) { instance_double(Messages::Creator) }
-
-      before do
-        allow(Messages::Creator).to receive(:new).and_return(messages_creator)
-        allow(messages_creator).to receive(:call).and_raise(Messages::CreationFailed, 'Error message')
-        message_resolver.execute_action
-      end
-
-      it 'sets error content' do
-        expect(message_resolver.content).to eq('⚠️ Could not create message: Error message')
-      end
 
-      it 'still sets the callback' do
-        expect(message_resolver.callback).to be_an_instance_of(DiscordEngine::InteractionCallback)
-      end
-
-      it 'sets empty components array' do
-        expect(message_resolver.components).to eq([])
+      it 'creates a DiscordMessage record' do
+        expect(DiscordMessage.count).to eq(1)
       end
+    end
 
-      it 'sets ephemeral flag' do
-        expect(message_resolver.flags).to eq(DiscordEngine::Message::EPHEMERAL_FLAG)
-      end
+    describe 'message creation fails' do
```

Comment:
This `describe` isn't necessary, let's remove it and move all the inner contexs out of it please

Reply:
Done!